samples/helloworld/README.md

```diff
@@ -8,7 +8,12 @@
 - Demonstrate how to optionally apply LVI mitigation to enclave code
 - Also runs in OE simulation mode
 
-Prerequisite: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+## Prerequisites
+
+- Use a OE SDK-supported machine or development environment (like Intel SGX).
+- Install the OE SDK package and dependencies for your environment. [Start here](/../../../openenclave#getting-started) to find OS-specific installation instructions. 
```

Comment:
Note that link text should be descriptive to aid accessibility tooling. How about:

> Install the OE SDK package and dependencies for your environment. See the [Getting Started documentation](/../../../openenclave#getting-started) to find OS-specific installation instructions.

Reply:
agreed. Fixed!